src/block/parser.rs

```diff
@@ -123,7 +123,7 @@ impl BlockParser {
                 orphaned_paths: vec![].into_iter(),
             })
         } else {
-            Ok(Self::empty(blocks_dir, &vec![]))
+            Ok(Self::empty(blocks_dir, &[]))
```

Comment:
I don't know enough to be able to tell if this change is related to the Rust minor version change. Is it?

Reply:
There are new lints when upgrading to the latest stable